core/runtime/binaryen/runtime_api/babe_api_impl.cpp

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "runtime/binaryen/runtime_api/babe_api_impl.hpp"
+
+namespace kagome::runtime::binaryen {
+
+  BabeApiImpl::BabeApiImpl(
+      const std::shared_ptr<runtime::WasmProvider> &wasm_provider,
+      const std::shared_ptr<extensions::ExtensionFactory> &extension_factory)
+      : RuntimeApi(wasm_provider, extension_factory) {}
```

Comment:
BOOST_ASSERTS that args are not null?

Reply:
There are no private fields in BabeApiImpl. Although there are no checks in parent class RuntimeApi. Included asserts there